app/frontend/wallet/shelley/shelley-bitbox02-crypto-provider.ts

```diff
@@ -224,7 +135,7 @@ const ShelleyBitBox02CryptoProvider = async ({
     return orderedTokenBundle.map(({policyId, assets}) => {
       const tokens = assets.map(({assetName, quantity}) => ({
         assetName: Buffer.from(assetName, 'hex'),
-        value: quantity.toString(),
+        value: quantity.toString() as unknown as bigint,
```

Comment:
We have these `as unknown as bigint` assertions, exactly 6 occurences in this file.

The reason is, the values (in this case it's `txAux.fee`) are of type "BigNumber" which [comes from a package called bignumber.js](https://mikemcl.github.io/bignumber.js/) and we need to somehow assert its type to be `bigint` to work with the current `bitbox-api`.

If `bitbox-api` allows `bigint | string` for these, then we can remove the assertions.

Reply:
More important Q: does `bitbox-api` support stringified values here (and in other places where we did similar)? 

cc @benma

Reply:
Note: this was one of the causes [of the trx error](https://github.com/shonsirsha/adalite/pull/1#pullrequestreview-2099065064).

Reply:
@shonsirsha it does not support `string`, just `BigInt`. Imo we should keep it this way, as BigInt has a clear type, while with strings, one can't be sure which kinds of formats are accepted - like `0x...` for hex (common in ETH).